pages/community/index.md

```diff
@@ -2,36 +2,28 @@
 layout: default
 ---
 
-# Contact
+# Getting and keeping in touch
 
-## Getting and keeping in touch
-
-### Mailing lists
-![LFEnergy](img/index/logo_lfenergy.png){: width="250" .center-image}
-
-Discussion and questions on PowSyBl's usage and development happens on the following mailing lists:
-- [General](https://lists.lfenergy.org/g/powsybl): General mailing list for user questions and discussions. This is also where new releases are announced.
-- [Developers](https://lists.lfenergy.org/g/powsybl-dev): Questions and discussions related to PowSyBl development.
-- [Technical Steering Committee](https://lists.lfenergy.org/g/powsybl-tsc): Votes and decisions impacting the project organization or its technical direction.
-
-### Slack
+## Slack
 ![Slack](img/index/slack-logo.png){: width="300" .center-image}
 
-We use the [PowSyBl Slack](https://join.slack.com/t/powsybl/shared_invite/zt-rzvbuzjk-nxi0boim1RKPS5PjieI0rA) organization for all chat. You can join to discuss or ask questions on PowSyBl's usage, if you suspect that you have found a bug in PowSyBl or if you'd like to propose an enhancement.
+We use the [PowSyBl Slack](https://join.slack.com/t/powsybl/shared_invite/zt-rzvbuzjk-nxi0boim1RKPS5PjieI0rA) organization for all chat. 
 
-## Reporting bugs
-<p style="text-align:center">
-  <img alt="Slack logo" src="img/index/slack-logo.png" width="300" />
-  <img alt="Github logo" src="img/index/github-logo.png" width="130" /> 
-</p>
+**This is the first place to come if you think you've found a bug in PowSyBl or if you want to suggest an enhancement**.
 
-If you encounter a problem with PowSyBl, the first places to ask for help are the user mailing list and the `#issues` [slack channel](https://powsybl.slack.com/archives/C02CG8Q0TS6).
+You can also join to discuss or ask any question about PowSyBl.
 
```

Comment:
Maybe we can add that user stories can be shared too, it is always a pleasure for the maintainers team!